zeus/monitor/energy.py

```diff
@@ -160,6 +160,7 @@ def __init__(
         cpu_indices: list[int] | None = None,
         approx_instant_energy: bool = False,
         log_file: str | Path | None = None,
+        backend: str = "torch",
```

Comment:
Throughout the changes, this should be typed as `Literal["torch", "jax"]`. `Literal` can be imported from `typing`.